modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/common/RESTAPIServerTestBase.java

```diff
@@ -40,4 +40,11 @@ protected void testInit(String apiVersion, String apiDefinition, String tenantDo
                 tenantDomain, apiVersion);
         super.init(apiDefinition, basePathInSwagger, basePath);
     }
+
+    protected void testInitWithoutTenantQualifiedPath(String apiVersion, String apiDefinition)
```

Comment:
what's the usage of `testInit` now ?

Reply:
It is used by other REST API test cases. Only tenant management rest API has been updated to not to accept tenant qualified endpoints